src/index.ts

```diff
@@ -12,9 +12,12 @@ import {
   translationImport
 } from "./commands";
 import type { BaseProps } from "./commands/npmCredentials/utils";
-import wefitLogo from "./constants/wefitLogo";
+import { makeWeFitLogo } from "./factories/makeWeFitLogo";
+import packageJson from "../package.json"
 
-console.log(chalk.yellow(wefitLogo));
+const CLI_VERSION = packageJson.version || "1.0.0";
```

Comment:
Acho que deixar esse 1.0.0 não seria o correto, mas informar que não foi possível obter a versão correta.
Como um "Not Found"